content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,6 +17,14 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+Use `GITEA_CUSTOM` to define another folder instead. For example:
+
+```
+GITEA_CUSTOM=/srv/gitea/mycustom ./gitea web
+```
+
+**Note** that by default Gitea will use the `custom` folder in the current working directory.
```

Comment:
I'm afraid at the moment it'll be the `custom` folder in the directory containing the `gitea` binary unless `GITEA_WORK_DIR` env is set, in which case that's the directory used to resolve relative paths like `custom/`